### PR TITLE
[Feature] Add option to display actions column as dropdown

### PR DESCRIPTION
**Pull request by maurohmartinez**
### BEFORE - What was wrong? What was happening before this PR?

Nothing! [Here](https://github.com/Laravel-Backpack/CRUD/issues/4963) it is well explained.

### AFTER - What is happening after this PR?

Basically... this!

actions-column-as-dropdown-button

### How did you achieve that, in technical terms?

Most changes are done in js, `src/resources/views/crud/inc/datatables_logic.blade.php`. A new operation setting is added in order to control it `actionsColumnAsDropdown`, since there might be people nostalgic about the old ways... :)

```
$this->crud->displayActionsColumnAsDropdown(false);
```

### Is it a breaking change?

Nope.

### How can we test the before & after?

Checkout this branch and test if columns can now be displayed as dropdown. Also, `$this->crud->displayActionsColumnAsDropdown(false);` should revert it to the old line stack.

### CODE REVIEW

**Comment by tabacitu** [comment]
I love this! ❤️ This is a GREAT solution Mauro - can't belive how simple it is using JS. 

The naming is good too. I took another look, it makes sense it like this because it's similar to other configs we have.

The only thing I'd change is the default. Let's default to `false`. If that's done, and you've tested this on all themes, I'll happily test and merge this to v6 🎉

File: src/config/backpack/operations/list.php [hunk 1]
```diff
@@ -46,6 +46,9 @@
     // - 4 - less important than most columns
     'actionsColumnPriority' => 1,
 
+    // Nest action buttons within a dropdown in actions column
+    'actionsColumnAsDropdown' => true,
```
Comment (tabacitu) [inline src/config/backpack/operations/list.php:50]:
```suggestion
    'actionsColumnAsDropdown' => false,
```

**Review by tabacitu — CHANGES_REQUESTED** [review]

**Comment by maurohmartinez** [comment]
@tabacitu tested it with other templates, and it looks good!

I made one small change to common.css (not related to this), but also tested that in other themes and it works good.

**Comment by maurohmartinez** [comment]
Close in favor of https://github.com/Laravel-Backpack/CRUD/pull/5005

**Pull request closed without merging**